projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -1430,6 +1430,31 @@ function wpcom_launchpad_mark_setup_site_tasks_complete( $old_value, $value ) {
 add_action( 'update_option_site_icon', 'wpcom_launchpad_mark_setup_site_tasks_complete', 10, 3 );
 add_action( 'update_option_site_logo', 'wpcom_launchpad_mark_setup_site_tasks_complete', 10, 3 );
 
+/**
+ * Mark the setup_general task as complete if the site title or the site description is changed.
+ *
+ * @param string $old_value The old value of the site title.
+ * @param string $value The new value of the site title.
+ *
+ * @return void
+ */
+function wpcom_launchpad_mark_setup_general_task_complete( $old_value, $value ) {
+	if ( defined( 'HEADSTART' ) && HEADSTART ) {
+		return;
+	}
+
+	if ( wp_installing() ) {
+		return;
+	}
+
+	l( 'CALLING wpcom_launchpad_mark_setup_site_tasks_complete', $old_value, $value );
```

Comment:
Did you intend to leave a log statement here? Worried we'll pollute the error logs with unnecessary notifications.

Reply:
nope, not intended at all. I will remove it. Thanks for spotting :)